game_eggs/steamcmd_servers/ark_survival_ascended/egg-ark--survival-ascended.json

```diff
@@ -62,9 +62,9 @@
         },
         {
             "name": "Battle Eye",
-            "description": "Enable BattleEye \/ Anti-Cheat",
```

Comment:
Please leave the comment and default value as they do not even see the 0 or 1 they see a toggle switch

Reply:
This is my first pr so help me out please.

I downloaded the changes you made to the branch, but how to continue ?

Do i make adjustments to the files in my branch in VS ? 
Or do i need to upload the json in the panel and export it ?
I'm confused ?

Reply:
you can not touch the .json file with an editor, you must make the changes in a panel and then hit export!